### PR TITLE
interfaces: fix interface handling on no-app snaps

### PR DESCRIPTION
**Pull request by niemeyer**

### CODE REVIEW

File: interfaces/repo.go [hunk 1]
```diff
@@ -470,25 +488,15 @@ func (r *Repository) securitySnippetsForSnap(snapName string, securitySystem Sec
...
 				return nil, err
 			}
-			if snippet == nil {
-				continue
-			}
-			for appName := range slot.Apps {
-				securityTag := snap.AppSecurityTag(snapName, appName)
-				snippets[securityTag] = append(snippets[securityTag], snippet)
-			}
+			addSnippet(snapName, slot.Name, slot.Apps, nil, snippets, snippet)
```
Comment (kyrofa) [inline interfaces/repo.go]: Should this be using the plug?

Comment (kyrofa) [inline interfaces/repo.go]: Ah, the snippet comes from the plug.

Comment (niemeyer) [inline interfaces/repo.go]: Right, it comes from the snap slot as far as this side of the connection is concerned.

**Comment by kyrofa** [comment]
:+1: looks good.

**Pull request merged**